test/fixtures/test-runner/bail/error.js

```diff
@@ -0,0 +1,12 @@
+const assert = require('assert');
+const common = require('../../../common');
```

Comment:
This file also needs strict mode. Can you also move common before everything else.